sirepo/template/radia_tk.py

```diff
@@ -1,9 +1,12 @@
+
 import numpy
 import radia
 import re
 import sirepo.util
 import sys
 
+from mpi4py import MPI
```

Comment:
what if there's no MPI. The whole "check for MPI" thing that's in sirepo.mpi is mooted by this. Don't import unqualified.

Reply:
20e2ea5

```import MPI``` was left over from messing around with timing the solve, but point taken